app/views/apps/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with url: "/apps", method: :post, data: {turbo: false}, local: true do |form| %>
+ <p> <%= form.label :name %>
+  <%= form.text_field :name, placeholder: "ex. Donky Kong" %></p>
+  <p><%= form.label :address %>
+  <%= form.text_field :address, placeholder: "ex. 123 Drewry lane"%></p>
+  <p><%= form.label :city %>
+  <%= form.text_field :city, placeholder: "ex. Denver"%></p>
+  <p><%= form.label :state %>
+  <%= form.text_field :state, placeholder: "ex. CO"%></p>
+  <p><%= form.label :zip %>
+  <%= form.text_field :zip, placeholder: "ex. 12345"%></p>
+  <p><%= form.label :description %>
+  <%= form.text_field :description, placeholder: "Why are you good home?"%></p>
```

Comment:
re: your question in your video: You could pass a local variable that's a boolean (like `show_description`) and then use an if statement with that boolean in the partial to decide whether to include that field. If there's a lot of variability with what fields should should up, though, I probably wouldn't use a partial